platform/wm/rhodes/MainWindow.cpp

```diff
@@ -1241,8 +1241,11 @@ LRESULT CMainWindow::OnExecuteJSCommand(WORD /*wNotifyCode*/, WORD /*wID*/, HWND
 {
     TNavigateData* nd = (TNavigateData*)hWndCtl;
     if (nd) {
-        LPTSTR wcurl = (LPTSTR)(nd->url);
-        if (wcurl) 
+        wchar_t wcurl[2046];
+	memset(wcurl,'0',2046);
```

Comment:
And yes, we do have javascript commands that are > 2046 characters potentially.  One example I know of is the 2.2 Version API but I'm sure it's a limitation will frequently be exceeded.